.goreleaser.yml

```diff
@@ -8,14 +8,23 @@ builds:
     goarch:
       - amd64
       - 386
+      - arm
       - arm64
       - s390x
       - ppc64le
+    goarm:
+      - 5
+      - 6
+      - 7
```

Comment:
do we need all 3? just thinking about keeping the number of permutations reasonable

Reply:
I only need armv6, but I think armv7 is also quite popular.
This should add exactly 3 builds: armv5, armv6 and armv7 for linux.